src/main/scala/scorex/core/app/Application.scala

```diff
@@ -63,7 +64,7 @@ trait Application extends ScorexLogging {
     */
   val swaggerConfig: String
 
-  val timeProvider = new NetworkTimeProvider(settings.ntp)
+  val timeProvider = new NetworkTimeProvider(settings.ntp)(executionContext)
```

Comment:
this is the only place, where we pass executionContext explicitly, other usages are implicit.